app/helpers/login.ts

```diff
@@ -0,0 +1,19 @@
+import { Keypair } from "stellar-sdk";
+
+function isSecretKeyValid(secretKey: string): boolean {
+  try {
+    Keypair.fromSecret(secretKey);
+    return true;
+  } catch (error) {
+    return false;
```

Comment:
Also, we're not throwing the error here.

Reply:
I added a throw error statement to handle this exception and removed the return